libs/sdk-ui-dashboard/src/model/commands/ui.ts

```diff
@@ -23,17 +24,61 @@ export interface ChangeRenderMode extends IDashboardCommand {
  * Creates the ChangeRenderMode command. Dispatching this command will result in change of the render mode of dashboard component
  *
  * @param renderMode - render mode value
+ * @param resetDashboard - determines whether the dashboard is reset when the render mode is changed
  * @param correlationId - specify correlation id to use for this command. this will be included in all
  *  events that will be emitted during the command processing
  *
  * @internal
  */
-export function changeRenderMode(renderMode: RenderMode, correlationId?: string): ChangeRenderMode {
+export function changeRenderMode(
+    renderMode: RenderMode,
+    resetDashboard = true,
+    correlationId?: string,
+): ChangeRenderMode {
     return {
         type: "GDC.DASH/CMD.CHANGE_RENDER_MODE",
         correlationId,
         payload: {
             renderMode,
+            resetDashboard,
+        },
+    };
+}
+
+/**
+ * Creates the ChangeRenderMode command for switch to edit mode.
+ *
+ * @param correlationId - specify correlation id to use for this command. this will be included in all
+ *  events that will be emitted during the command processing
+ *
+ * @internal
+ */
+export function switchToEditRenderMode(correlationId?: string): ChangeRenderMode {
+    return {
```

Comment:
Should we re-use `changeRenderMode` function in the function body for these specific commands, to DRY it?